libraries/app/database_api.cpp

```diff
@@ -624,6 +625,89 @@ vector<optional<account_object>> database_api_impl::get_accounts(const vector<ac
    return result;
 }
 
+vector<limit_order_object> database_api::get_account_limit_orders( const string& name_or_id, const string &base, const string &quote, uint32_t limit, optional<limit_order_id_type> ostart_id, optional<price> ostart_price)
+{
+   return my->get_account_limit_orders( name_or_id, base, quote, limit, ostart_id, ostart_price );
+}
+
+vector<limit_order_object> database_api_impl::get_account_limit_orders( const string& name_or_id, const string &base, const string &quote, uint32_t limit, optional<limit_order_id_type> ostart_id, optional<price> ostart_price)
+{
+   FC_ASSERT( limit <= 100 );
```

Comment:
ok, but why 101 easier for pagination?

Reply:
Usually we use 100, 50 or etc as page size. Since we use `lower_bound` here, if pass in the last item of last page, with `100` as limit we can get only 99 new items (although it's possible to get around it e.g. pass in `last_id+1` but it's not convenient).